designer/server/src/main/scala/pl/touk/nussknacker/ui/statistics/UsageStatisticsReportsSettingsDeterminer.scala

```diff
@@ -36,9 +44,13 @@ object UsageStatisticsReportsSettingsDeterminer {
       processService: ProcessService,
       // TODO: Instead of passing deploymentManagerTypes next to processService, we should split domain ScenarioWithDetails from DTOs - see comment in ScenarioWithDetails
       deploymentManagerTypes: ProcessingTypeDataProvider[DeploymentManagerType, _],
-      fingerprintSupplier: (UsageStatisticsReportsConfig, FileName) => Future[Fingerprint]
+      fingerprintSupplier: (
```

Comment:
WDYT about refactoring it a little bit?
In this case, the supplier is nothing but FignerprintService. Why cannot we use it here?

Reply:
Done